sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2Suites.scala

```diff
@@ -493,12 +493,12 @@ class HiveThriftBinaryServerSuite extends HiveThriftJdbcTest {
       val jarPath = "../hive/src/test/resources/TestUDTF.jar"
       val jarURL = s"file://${System.getProperty("user.dir")}/$jarPath"
 
-      Seq(
-        s"ADD JAR $jarURL",
+      statement.execute(s"ADD JAR $jarURL")
+      statement.execute(
         s"""CREATE TEMPORARY FUNCTION udtf_count2
            |AS 'org.apache.spark.sql.hive.execution.GenericUDTFCount2'
          """.stripMargin
-      ).foreach(statement.execute)
+      )
```

Comment:
When the original code fails, log lines only say that one of the commands within the `Seq` fails. I'd like to know which one fails.